src/promptflow/tests/sdk_cli_test/conftest.py

```diff
@@ -69,11 +73,23 @@ def local_custom_connection(local_client, azure_open_ai_connection):
 
 
 @pytest.fixture
-def setup_local_connection(local_client):
+def setup_local_connection(local_client, azure_open_ai_connection):
     global _connection_setup
     if _connection_setup:
         return
     connection_dict = json.loads(open(CONNECTION_FILE, "r").read())
+    if RecordStorage.is_replaying_mode():
```

Comment:
is this required? if we don't do this, will aoai connection not in the connection dict?

Reply:
Yes

I won't create this CONNECTION_FILE in the new pipeline, so that the connection_dict is empty in line 80.